### PR TITLE
wiso-steuer-2022

### PR DESCRIPTION
**Pull request by piijai**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

update cask as suggested in the latest pull request for wiso-steuer-2024
https://github.com/Homebrew/homebrew-cask/pull/181821#pullrequestreview-2225852218

### CODE REVIEW

**Comment by khipp** [comment]
Thank you, @piijai. There were some additional changes made before https://github.com/Homebrew/homebrew-cask/pull/181821 was merged. This PR and the one for `wiso-steuer-2023` will have to be updated accordingly.

**Comment by piijai** [comment]
Updated cask according to https://github.com/Homebrew/homebrew-cask/pull/181821

File: Casks/w/wiso-steuer-2022.rb [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
+      match[1]
+    end
   end
 
   auto_updates true
   depends_on macos: ">= :mojave"
 
-  app "SteuerMac 2022.app"
+  # Renamed for consistency: app name differs in Finder to shell
+  app "SteuerMac 2022.app", target: "WISO Steuer-Mac 2022.app"
```
Comment (khipp) [inline Casks/w/wiso-steuer-2022.rb:27]:
```suggestion
  app "SteuerMac 2022.app", target: "WISO Steuer 2022.app"
```

We probably should remove the `-Mac` part to be consistent with the other cask (`WISO Steuer 2023.app` and `WISO Steuer 2024.app`).

Comment (piijai) [reply Casks/w/wiso-steuer-2022.rb:27]: I downloaded the dmg file and compared the app names and it's "WISO Steuer-Mac 2022.app" for 2022. Starting from 2023 the app name changed to "WISO Steuer 2023.app".
What way do you prefer? Consistend names vs. the app names, how they are provided in the dmg files.

Comment (khipp) [reply Casks/w/wiso-steuer-2022.rb:27]: On second thought, you are right. It should be consistent with what is shown in the Finder (`WISO Steuer-Mac 2022.app`).

**Review by khipp — APPROVED** [review]
Thank you!

**Pull request merged**